pkg/cmd/upgrade/upgrade_extensions.go

```diff
@@ -1,381 +0,0 @@
-package upgrade
-
-import (
-	"bytes"
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"os/user"
-	"path/filepath"
-	"strings"
-	"time"
-
-	"github.com/jenkins-x/jx/pkg/cmd/create/options"
-
-	"github.com/jenkins-x/jx/pkg/cmd/helper"
-
-	"github.com/jenkins-x/jx/pkg/extensions"
-
-	"github.com/pkg/errors"
-
-	"github.com/blang/semver"
-	"github.com/jenkins-x/jx/pkg/kube"
-
-	"github.com/jenkins-x/jx/pkg/util"
-
-	"github.com/jenkins-x/jx/pkg/log"
-
-	typev1 "github.com/jenkins-x/jx/pkg/client/clientset/versioned/typed/jenkins.io/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	jenkinsv1 "github.com/jenkins-x/jx/pkg/apis/jenkins.io/v1"
-
-	"github.com/ghodss/yaml"
-
-	"github.com/jenkins-x/jx/pkg/cmd/opts"
-	"github.com/jenkins-x/jx/pkg/cmd/templates"
-	"github.com/spf13/cobra"
-)
-
-//const upstreamExtensionsRepositoryGitHub = "https://raw.githubusercontent.com/jenkins-x/jenkins-x-extensions/master/jenkins-x-extensions-repository.lock.yaml"
-const upstreamExtensionsRepositoryGitHub = "github.com/jenkins-x/jenkins-x-extensions"
-
-var (
-	upgradeExtensionsLong = templates.LongDesc(`
-		Upgrades the Jenkins X extensions available to this Jenkins X install if there are new versions available
-`)
-
-	upgradeExtensionsExample = templates.Examples(`
-		
-		# upgrade extensions
-		jx upgrade extensions
-
-	`)
-)
-
-type UpgradeExtensionsOptions struct {
```

Comment:
@abayer, @daveconde, @dgozalo, @ccojocar - is it safe to remove extension-related stuff? There are no "extensions" in use anymore, right?